news/2018-06-02-mappers-choice-awards-2017-results.md

```diff
@@ -0,0 +1,345 @@
+
+*written by [Tofu1222](https://osu.ppy.sh/u/6089608)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/hitorigoto.jpg)](https://osu.ppy.sh/s/602861)
+
+Definitely one of the most popular song just by looking at the amount of plays that its standard counterpart has, and it's clearly not the exception for mania either. **[Tofu1222](https://osu.ppy.sh/u/6089608)** manages to bring such an unique touch with the mixing of the piano keysound that further develop the players enjoyment, and not stopping on that, he also uniquely bring his more unorthodox style of pattern approachs in the anime map by putting vocal pitch emphasis into kiai section by layering it into jackhammer. With multiple plus factor that he put into the map, he manages to bring himself into the top spot for the japanese category.
+
+*written by [Maxus](https://osu.ppy.sh/u/4335785)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/dragonforce.jpg)](https://osu.ppy.sh/s/584941)
+
+Being famous for making DragonForce maps, **[Surono](https://osu.ppy.sh/users/3611370)** brought us with a great osu!mania creation of DragonForce marathon in 2017. This map definitely makes up a training for stamina, with jumpstreams of 190 bpms mapped throughout the song, but what's more, its variated patterns in vocal part and guitar solo part marks a noticeable characteristic of this map, thus making it a great fit of Western category.
+
+*written by [Tofu1222](https://osu.ppy.sh/u/6089608)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/garalulu.jpg)](https://osu.ppy.sh/u/757783)
+
+**[Garalulu](https://osu.ppy.sh/u/757783)** is making huge progress in both mapping and modding last year. With new 7K and 4K maps ranked, he showed us his overwhelming experience in creating maps of different key modes, manipulating SVs, and using different types of advanced patterns such as LN manipulation. Furthermore, his great modding skill helped him become a BN at the end of last year, which marks his knowledge in modding as well. That's why he can have the strength to take the Grand User Award for this mode.
+
+*written by [Tofu1222](https://osu.ppy.sh/u/6089608)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/kawawa.jpg)](https://osu.ppy.sh/u/4647754)
+
+As one of the oldest mapper and modder that is still active nowadays, **[Kawawa](https://osu.ppy.sh/u/4647754)** contributed many great 7K mapsets. Those sets with spreads provide the advancing players with increasing levels of practices, while the marathons, featured with delicate hitsounds and variated patterns, are loved by those seeking for challenges. It is of no surprise that his deep experience in mapping won him osu!mania mapper category.
```

Comment:
mapper and modder -> mappers and modders
(probably)
practices -> difficulty
him osu!mania -> him the osu!mania